css/fiona-pham.css

```diff
@@ -0,0 +1,20 @@
+body {
+  font-family: sans-serif;
+  line-height: sensible;
```

Comment:
_Sensible_ is not a valid value for the `line-height` property. Check the documentation [here](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height).